### PR TITLE
Refactored function

### PR DESCRIPTION
**Pull request by AnthonyLaw**
### Added
- getDiagnostic sdk function
- getmarketData api function
- chainInfo in chain store
- marketData in chain store
- marketData endpoint url

### CODE REVIEW

**Comment by Alexhuszagh** [comment]
@AnthonyLaw Can you rebase it off the current master? This way, your commits don't include duplicated work.

File: client/src/components/NemPriceGraph.vue [hunk 1]
```diff
@@ -66,137 +65,57 @@
...
-    width: 100%;
+  margin: 0px 0px 0px -10px;
+  width: 100%;
 }
 </style>
 <script>
-import axios from 'axios'
+import { mapGetters } from 'vuex'
+import w1 from '@/components/Chart.vue'
 import ApexCharts from 'apexcharts'
```
Comment (OlegMakarenko) [inline client/src/components/NemPriceGraph.vue]: We don't need this import anymore

File: client/src/components/NemPriceGraph.vue [hunk 2]
```diff
@@ -66,137 +65,57 @@
...
-          axisBorder: {
-            show: false,
-            color: '#0998a6'
-          }
-        },
-        tooltip: {
-          enabled: true
-        }
+      if (this.marketData.historicalHourlyGraph.Data) {
+        this.marketData.historicalHourlyGraph.Data.forEach((item, index) => {
```
Comment (OlegMakarenko) [inline client/src/components/NemPriceGraph.vue]: I think it is better to move this part to the Store

File: client/src/components/NemPriceGraph.vue (hunk 1 above)
Comment (OlegMakarenko) [inline client/src/components/NemPriceGraph.vue]: We don't need this import anymore

File: client/src/components/NemPriceGraph.vue (hunk 2 above)
Comment (AnthonyLaw) [reply client/src/components/NemPriceGraph.vue]: will move to store :)

File: client/src/components/NemPriceGraph.vue (hunk 1 above)
Comment (AnthonyLaw) [reply client/src/components/NemPriceGraph.vue]: thanks i will remove it!

**Pull request merged**